website/src/get-started/wordcount-example.md

```diff
@@ -1344,7 +1356,11 @@ PCollection<String> windowedWords = input
 ```
 
 ```py
-# This feature is not yet available in the Beam SDK for Python.
+import apache_beam as beam
+from apache_beam.transforms import window
+...
+...
+windowed_words = input | beam.WindowInto(window.FixedWindows(60 * window_size_min))
```

Comment:
For uniformity, lets rename `window_size_min` to `window_size_minutes `

Reply:
make sense. I will update it.